dataeng/jobs/analytics/DBTRun.groovy

```diff
@@ -77,8 +81,12 @@ class DBTRun{
                 env('DBT_TARGET', allVars.get('DBT_TARGET'))
                 env('DBT_PROFILE', allVars.get('DBT_PROFILE'))
                 env('DBT_PROJECT_PATH', 'automated/applications')
-                env('DBT_RUN_OPTIONS', allVars.get('DBT_RUN_OPTIONS'))
-                env('DBT_RUN_EXCLUDE', allVars.get('DBT_RUN_EXCLUDE'))
+                env('DBT_MODEL_INCLUDE', allVars.get('DBT_MODEL_INCLUDE'))
+                env('DBT_MODEL_EXCLUDE', allVars.get('DBT_MODEL_EXCLUDE'))
+                env('FULL_REFRESH', allVars.get('FULL_REFRESH'))
+                env('DBT_RUN_ARGS', allVars.get('DBT_RUN_ARGS'))
+                env('SKIP_TESTS', false)
+                env('DBT_TEST_ARGS', allVars.get('DBT_TEST_ARGS'))
```

Comment:
What is this dbt-automated job, exactly?  According to the description, it runs after schema builder PRs are merged?  The trigger runs on a cron schedule, not based on a git commit.  Is this what takes the place of us having to run dbt manually after merging a schema builder PR?  Why is it a separate job than dbt-manual?

Sorry, I just have so many questions about this other job.

Reply:
Historically, we have used the `dbt-manual` job for two purposes:

1. Regularly running the models built by dbt-schema-builder after a PR is merged.
2. Allowing other teams to run data models from the `reporting` project in an ad-hoc fashion against production.

It seemed like we could separate these concerns and the former could be automated.

As for the trigger, this is a result of our Jenkins configuration. Since it is not publicly accessible, it can't receive webhooks from Github. Instead, we poll Github regularly to check for new commits. If new commits are detected, it checks to see if they were created by dbt-schema-builder before proceeding.